app/presenters/rejection_reasons/rejection_reasons_presenter.rb

```diff
@@ -67,7 +67,7 @@ def tailored_advice_reasons
 
     def render_tailored_advice_section_headings?
       # Only render headings if there are multiple reasons given
-      tailored_advice_reasons.keys.many? || tailored_advice_reasons.values.first&.many?
+      tailored_advice_reasons.keys.many? || (tailored_advice_reasons.values.first.instance_of?(Array) && tailored_advice_reasons.values.first.many?)
```

Comment:
`Array(tailored_advice_reasons.values.first).many?` would work? Happy as it is.

Reply:
I'll make this nicer and add tests once we get it in -- it's affected 65 people (all from the same institution, so from the api) so far, so want to put it in asap.